### PR TITLE
don't require cloning neutron/lbaas for unit tests

### PR DESCRIPTION
**Pull request by dougwig**

### CODE REVIEW

File: a10_neutron_lbaas/v2/neutron_ops.py [hunk 1]
```diff
@@ -45,3 +49,6 @@ def loadbalancer_total(self, context, tenant_id):
 
     def pool_get(self, context, pool_id):
         return self.plugin.db.get_pool(context, pool_id)
+
+    def bcm_factory(self):
+        return bcm
```
Comment (dougwig) [inline a10_neutron_lbaas/v2/neutron_ops.py]: bug, this line should be "return bcm.CertManager()"

**Comment by dougwig** [comment]
check

**Comment by dougwig** [comment]
check

**Comment by Cedev** [comment]
The Jenkins test failures are that it couldn't get a mirror for the lbaasv1 4.0.3 failure and something inscrutable for the lbaasv1 272 failure. Does the `bcm_factory` bug need to be addressed before merging?

File: a10_neutron_lbaas/v2/neutron_ops.py (hunk 1 above)
Comment (mdurrant-b3) [inline a10_neutron_lbaas/v2/neutron_ops.py]: `bcm_factory` is called as such:
https://github.com/a10networks/a10-neutron-lbaas/pull/269/files#diff-d79997eae33bb0fff59596d6b9a52cbbR21
So yes, yes it should.

**Comment by mdurrant-b3** [comment]
lgmt :+1:

**Pull request merged**